cryptocurrency/README.md

```diff
@@ -2,23 +2,57 @@
 
 The script will pull live and historical data for specified cryptocurencies from the [CoinGecko](https://www.coingecko.com/) website into [Deephaven](https://github.com/deephaven/deephaven-core).
 
+
+## How it works
+
+### Deephaven application mode
+
+This app runs using [Deephaven's application mode](https://deephaven.io/core/docs/how-to-guides/app-mode/).
+
+### Components
+* `Dockerfile` - The dockerfile for the application. This extends the default Deephaven image to add dependencies. See our guide, [How to install Python packages](https://deephaven.io/core/docs/how-to-guides/install-python-packages/#add-packages-to-a-custom-docker-image), for more information.
+* `docker-compose.yml` - The Docker Compose file for the application. This is mostly the same as the [Deephaven docker-compose file](https://raw.githubusercontent.com/deephaven/deephaven-core/main/containers/python-examples/docker-compose.yml) with modifications to run in application mode.
+* `start.sh` - A simple helper script to launch the application.
+* `app.d/cryptoApp.app` - The Deephaven application mode app file.
+* `app.d/crypto.py` - The Python script that pulls the data from [CoinGecko](https://www.coingecko.com/) and stores it into Deephaven.
+* `crypto.png` - Image of the end state of Deephaven tables.
+
+
+### High level overview
+
+This app pulls data from  [CoinGecko](https://www.coingecko.com/) through HTTP requests. The API responses are deserialized, and the desired values are extracted and stored into a Deephaven table.
+
+Once data is collected and tables are created, various [Deephaven queries](https://deephaven.io/core/docs/how-to-guides/simple-python-query/) are then performed on the tables.
```

Comment:
```suggestion
Once data is collected and tables are created, various Deephaven queries are then performed on the tables.
```

Reply:
IMO, we do not need the link on the phrase. But how about, "Please see our [full documentation](**link**) to learn about the Deephaven Query Language.